test/object-store/realm.cpp

```diff
@@ -4173,3 +4176,165 @@ TEST_CASE("KeyPathMapping generation") {
         REQUIRE(q.count() == 0);
     }
 }
+
+#if REALM_HAVE_UV
+namespace realm::util {
+class UvLoopScheduler final : public util::Scheduler {
```

Comment:
Please, do rewrite this test before merging to be scheduler agnostic so that we could verify our abstractions for Scheduler and EventLoop as they are supposed to be used on all platforms with sdks.

Reply:
I am not sure that can be done. Our UV scheduler only supports one thread (As I understand it). But perhaps we don't need different threads. I will investigate that.

Reply:
I have now rewritten the test case. It was not required to create a second thread. Just a second realm.